base_product_mass_addition/models/product_product.py

```diff
@@ -13,25 +13,59 @@ class ProductProduct(models.Model):
 
     qty_to_process = fields.Float(
         compute="_compute_process_qty",
-        inverse="_inverse_set_process_qty",
+        inverse="_inverse_quick_vals",
         help="Set this quantity to create a new line "
         "for this product or update the existing one.",
     )
+    quick_uom_category_id = fields.Many2one(
+        "uom.category", compute="_compute_quick_uom_category_id"
+    )
+    quick_uom_id = fields.Many2one(
+        "uom.uom",
+        domain="[('category_id', '=', quick_uom_category_id)]",
+        compute="_compute_quick_uom_id",
+        inverse="_inverse_quick_vals",
+    )
+
+    def _inverse_quick_vals(self):
```

Comment:
Creating the method "_sync_quick_lines" seem useless as in "_inverse_quick_vals" we only call this method.
Why just direct put the code of the method "_sync_quick_lines" inside "_inverse_quick_vals"  ?

Reply:
Yes, removed.

I think I was having problems with qty_to_process and quick_uom_id having different inverse functions and switching things here and there, but now it is useless.